.github/workflows/auto-merge.yml

```
@@ -182,3 +172,326 @@ jobs:
+    if: needs.auto-merge.outputs.merged_count > 0 && (needs.auto-merge.outputs.classic_challenges != '' || needs.auto-merge.outputs.package_challenges != '')
+    runs-on: ubuntu-latest
+    
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+        with:
+          ref: main
+          fetch-depth: 1
+
+      - name: Set up Go
+        uses: actions/setup-go@v4
```

actionlint (1.7.12)

[error] the runner of "actions/setup-go@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)